src/types.ts

```diff
@@ -101,3 +101,40 @@ export interface Column {
   stylingInfo: string[];
   sortDirection: string;
 }
+
+export interface PageInfo {
+  page: number;
+  rowsPerPage: number;
+  rowsPerPageOptions: number[];
+}
+
+export type SetPageInfo = stardust.SetStateFn<{
+  page: number;
+  rowsPerPage: number;
+  rowsPerPageOptions: number[];
+}>;
+
+export type TableData = {
+  totalColumnCount: number;
+  totalRowCount: number;
+  totalPages: number;
+  paginationNeeded: boolean;
+  rows: Record<string, number>;
+  columns: number[];
```

Comment:
This is wrong. columns should be `Column[]`. If you don't want to do rows properly now set it to `Record<srting, any>` for now. Better to be loosely defined than incorrect

Reply:
Right, looked at handle-data and fixed in https://github.com/qlik-oss/sn-table/pull/637/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR17